lib/rubocop/cop/rspec/factory_bot/association_style.rb

```diff
@@ -0,0 +1,254 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module RSpec
+      module FactoryBot
+        # Use consistent style to define associations.
+        #
+        # @safety
+        #   This cop may cause false-positives in `EnforcedStyle: explicit`
+        #   case. It recognizes any method call that has no arguments as an
+        #   implicit association but it might be a user-defined trait call.
+        #
+        # @example EnforcedStyle: implicit (default)
+        #   # bad
+        #   association :user
+        #
+        #   # good
+        #   user
+        #
+        # @example EnforcedStyle: explicit
+        #   # bad
+        #   user
+        #
+        #   # good
+        #   association :user
+        #
+        #   # good (defined in NonImplicitAssociationMethodNames)
+        #   skip_create
```

Comment:
What do you think of using something that looks like an global sequence attribute call here, e.g. `email`?
```suggestion
        #   # good (defined in NonImplicitAssociationMethodNames)
        #   email
```

Reply:
Looks good 👍 
Besides, now that `skip_create` is set by constant instead of config, it is strange that this value is used as an example here.